src/components/Loader/Loader.css

```diff
@@ -0,0 +1,21 @@
+.loader {
+    margin: 200px auto;
+    border: 15px solid var(--DARK-GREY);
+    width: 150px;
+    height: 150px;
```

Comment:
using rem will be better

Reply:
exactly that is a Loader using rem can hinder its dimension and since its smaller than 250px
Its not gonna matter. So that's why it was kept like that